### ISSUE TITLE
[BUG] tezos-client doesn't see Ledger Nano S on Linux

### ISSUE THREAD

**Issue by itkach** [serokell/tezos-packaging#180]
# To Reproduce

- install tezos-client on a Linux machine (not VM)
- plug in Ledger Nano S into a USB port
- start tezos wallet or baking app on the Ledger device
- run `tezos-client list connected ledgers`
- observe no ledgers listed

# Expected behavior
- tezos-client finds and reports connected ledger with output that start with
```
## Ledger `mark-dog-light-monkey`
Found a Tezos Baking 2.2.9 (git-description: "b28c2364") application running
on Ledger Nano S at [0001:0007:00].
...
```
# Environment
- Raspberry Pi OS (64 Bit)
- 9.0-1

# Additional context
Looks like udev rules need to be created for the ledger device. Ledger has a repo for the rules: https://github.com/LedgerHQ/udev-rules. Also note currently proposed rules are reported to not work on Ubuntu 18.04, which also appears to be the case for the 64-bit Raspberry Pi OS: https://github.com/LedgerHQ/udev-rules/issues/5. As mentioned in the issue, older rules do work. I added these udev rules from Kiln: https://gitlab.com/tezos-kiln/kiln/-/blob/develop/linux-distros.nix#L171 and tezos-client started to see the device.

**Comment by rvem** [comment]
Cannot be reproduced on Ubuntu 18.04.5 :(

**Comment by rvem** [comment]
Btw, which firmware version your ledger has?

**Comment by rvem** [comment]
@itkach, btw, which firmware version your ledger has?

**Comment by itkach** [comment]
> @itkach, btw, which firmware version your ledger has? Mine has 1.6.1 along with Tezos Wallet 2.2.9, maybe there is a possibility that [LedgerHQ/udev-rules#5](https://github.com/LedgerHQ/udev-rules/issues/5) was fixed in the firmware 🤔

same, 1.6.1 with 2.2.9 tezos apps

**Comment by rvem** [comment]
Fixed in v9.1-2 native packages

**Comment by Getbook1** [comment]
> # fuck off 
> 
> - install tezos-client on a Linux machine (not VM)
> - plug in Ledger Nano S into a USB port
> - start tezos wallet or baking app on the Ledger device
> - run `tezos-client list connected ledgers`
> - observe no ledgers listed
> 
> # Expected behavior
> - tezos-client finds and reports connected ledger with output that start with
> ```
> ## Ledger `mark-dog-light-monkey`
> Found a Tezos Baking 2.2.9 (git-description: "b28c2364") application running
> on Ledger Nano S at [0001:0007:00].
> ...
> ```
> # Environment
> - Raspberry Pi OS (64 Bit)
> - 9.0-1
> 
> # Additional context
> Looks like udev rules need to be created for the ledger device. Ledger has a repo for the rules: https://github.com/LedgerHQ/udev-rules. Also note currently proposed rules are reported to not work on Ubuntu 18.04, which also appears to be the case for the 64-bit Raspberry Pi OS: https://github.com/LedgerHQ/udev-rules/issues/5. As mentioned in the issue, older rules do work. I added these udev rules from Kiln: https://gitlab.com/tezos-kiln/kiln/-/blob/develop/linux-distros.nix#L171 and tezos-client started to see the device.
> 
> 
>
